### PR TITLE
Add a banner 'under construciton'

### PR DESCRIPTION
**Pull request by gtca**
Address #13.

### CODE REVIEW

File: layouts/partials/header.html [hunk 1]
```diff
@@ -1,4 +1,8 @@
-<header>
+<div id="banner">
+	<div id="banne-content">
```
Comment (ivirshup) [inline layouts/partials/header.html]:
```suggestion
	<div id="banner-content">
```

?

**Comment by ivirshup** [comment]
image

Maybe a color or a border or something so it sticks out?

**Comment by gtca** [comment]
image

Looks like the [relevant](https://www.pantone.com/connect/RED-032-U) styles have not made it to the .css file while `hugo server` works just fine.

**Comment by ivirshup** [comment]
@gtca, it looks like it's retrieving the css file from `scverse.org` even when served locally. So it's not seeing an updated `css` file.

image

**Comment by ivirshup** [comment]
So just merging should work, but also it would be nice if the url was generated differently.

**Pull request merged**